src/barril/units/posc.py

```diff
@@ -13317,6 +13318,116 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
         f_unit_to_base,
         default_category=None,
     )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 1.0, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/Pa",
+        "degC/Pa",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 0.00001, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 0.00001, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/bar",
+        "degC/bar",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0e6, 0.0)
```

Comment:
should these be `MakeCustomaryToBase(0.0, 1.0, 1.0e-6, 0.0)`

Reply:
Thanks again! I think I'm misunderstanding `MakeCustomaryToBase` and `MakeBaseToCustomary`. Should conversion factors always be inverted between these functions?

Reply:
No, no, the conversion factors are the same, my comment applies to both lines, I should have used the new GitHub feature to mark more lines ;)